tools/tasks/build.html_css.prod.ts

```diff
@@ -17,7 +17,9 @@ export = function buildJSDev(gulp, plugins) {
 
     function minifyCss() {
       return gulp.src(join(APP_SRC, '**/*.css'))
-        .pipe(plugins.minifyCss())
+        .pipe(plugins.sourcemaps.init())
```

Comment:
AFAIK source maps are only requested when the dev tools are opened. I am happy to remove it though.